src/app/sitemap.ts

```diff
@@ -0,0 +1,111 @@
+import type { MetadataRoute } from "next";
+import type { CategoryType } from "@/lib/types/categories";
+
+export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
+  const apiURL = process.env.GRAPHQL_API_URL || "";
+  const baseURL = "https://volt-eosin.vercel.app";
```

Comment:
This should come from environment variable

Reply:
Will address this in next PR